src/signals.c

```diff
@@ -39,3 +40,21 @@ void	sigquit_handler_in_child(int sig)
 		ft_printf("Quit\n");
 	}
 }
+
+void	sigint_handler_after_here_doc(int sig)
```

Comment:
Isn't this just the same as siging_handler_in_parent?

Reply:
Difference is that I don't print a new line... I just copied my bash behavior. Let's check it out.